histomicstk/features/graycomatrixext.py

```diff
@@ -164,8 +164,7 @@ def graycomatrixext(im_input, im_roi_mask=None,
             neigh_valid[neigh_coord_ind[j] >= num_levels] = False
 
         for j in range(num_dims):
-            neigh_coord_ind[j] = np.compress(neigh_valid,
-                                             neigh_coord_ind[j],
+            neigh_coord_ind[j] = np.compress(neigh_valid, neigh_coord_ind[j],
                                              axis=0).astype(np.int64)
 
         neigh_lin_ind = np.ravel_multi_index(neigh_coord_ind,
```

Comment:
@cdeepakroy Fix a type error in graycomatrixext.py.  The type of neigh_coord_ind was float64.